src/ansys/units/base_dimensions.py

```diff
@@ -19,38 +19,46 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-"""Provides the ``BaseDimensions`` class."""
+"""
+Provides the ``BaseDimensions`` class.
 
-from enum import Enum
+Supplies all valid base dimensions used in dimensional analysis.
 
+Used as dictionary keys for defining a `Dimensions` object.
+"""
 
-class BaseDimensions(Enum):
-    """
-    Supplies all valid base dimensions used in dimensional analysis.
+from enum import Enum
+import os
 
-    Used as dictionary keys for defining a `Dimensions` object.
+_base_dims = {
+    "MASS": 0,
+    "LENGTH": 1,
+    "TIME": 2,
+    "TEMPERATURE": 3,
+    "TEMPERATURE_DIFFERENCE": 4,
+    "CHEMICAL_AMOUNT": 5,
+    "LIGHT": 6,
+    "CURRENT": 7,
+}
 
-    Attributes
-    ----------
-    MASS
-    LENGTH
-    TIME
-    TEMPERATURE
-    TEMPERATURE_DIFFERENCE
-    ANGLE
-    CHEMICAL_AMOUNT
-    LIGHT
-    CURRENT
-    SOLID_ANGLE
-    """
+_base_dims_with_angle = {
+    "MASS": 0,
+    "LENGTH": 1,
+    "TIME": 2,
+    "TEMPERATURE": 3,
+    "TEMPERATURE_DIFFERENCE": 4,
+    "CHEMICAL_AMOUNT": 5,
+    "LIGHT": 6,
+    "CURRENT": 7,
+    "ANGLE": 8,
+    "SOLID_ANGLE": 9,
+}
```

Comment:
We should eliminate duplication:
```suggestion
_base_dims_with_angle = {**_base_dims, "ANGLE": 8, "SOLID_ANGLE": 9}
```

Reply:
Done. Thank you.